### DISCUSSION TITLE
Create Order Draft with User that doesn't have Manage Order permission

### DISCUSSION THREAD

**Discussion by eric-makro** [saleor/saleor#10605] [Q&A]
I can create Order Draft with User that doesn't have Manage Order permission.
Is there anyone has same issue?

**Comment by peelar** [comment]
Hello @eric-makro 👋 

Thanks for your request. Can you please [follow the bug template](https://github.com/saleor/saleor/issues/new?assignees=&labels=bug%2Ctriage&template=bug_report.yaml&title=Bug%3A+), especially the steps to reproduce?

**Reply by eric-makro** [reply]
Sure.

What are you trying to achieve?
- Checking user group permission validation during create draft order

Steps to reproduce the problem:
1. I use user_id = 1 (email: admin@example.com) and set group_id = 6 (User management) only
   
<img width="1289" alt="image" src="https://user-images.githubusercontent.com/110385557/190298612-22061c2e-154e-4ddc-b21a-9206a8b25af9.png">

2. I request JWT token with this user email and password

<img width="1064" alt="image" src="https://user-images.githubusercontent.com/110385557/190298747-05b82480-ea6c-46f4-a1d0-6152bdfd445c.png">

3. I use this token to query GraphQL mutation draftOrderCreate

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/110385557/190298889-bfa5e239-baf1-48a8-8466-8d2cd74f0758.png">

<img width="1052" alt="image" src="https://user-images.githubusercontent.com/110385557/190298937-2a9a7cb1-ea53-45b9-8029-1b223c2a207b.png">

4. I can get the draft Order created, no permission error 
 
5. But, if I don't use token authentication, I can get the error ("You need one of the following permissions: MANAGE_ORDERS",) which is I think it should appear if on previous step.

<img width="1052" alt="image" src="https://user-images.githubusercontent.com/110385557/190299090-b75be2d8-f05c-4f7a-bba0-649f2327eaeb.png">

What did you expect to happen?
- If I login with user without MANAGE_ORDERS permission, it should has error notification ("You need one of the following permissions: MANAGE_ORDERS") during create order.

Thank you

**Reply by peelar** [reply]
Thank you @eric-makro, we need some more information from you to confirm whether it is a bug.

Can you please run this query:

```
query MyPermissionAndGroups{
  me{
    email
    userPermissions{
      code
    }
    permissionGroups{
      name
      permissions{
        code
      }
    }
  }
}
```

and share the result? It needs to be executed as an admin.
